src/parser/SparqlLexer.cpp

```diff
@@ -19,8 +20,8 @@ const std::string SparqlLexer::PN_CHARS_BASE =
     "[\\x{200C}-\\x{200D}]|[\\x{2070}-\\x{218F}]|[\\x{2C00}-\\x{2FEF}]|"
     "[\\x{3001}-\\x{D7FF}]|[\\x{F900}-\\x{FDCF}]|[\\x{FDF0}-\\x{FFFD}]|"
     "[\\x{10000}-\\x{EFFFF}]";
-const std::string SparqlLexer::WS = "(\\x20|\\x09|\\x0D|\\x0A)";
-const std::string SparqlLexer::ECHAR = "\\\\[tbnrf\"']";
+const std::string SparqlLexer::WS = R"((\x20|\x09|\x0D|\x0A))";
+const std::string SparqlLexer::ECHAR = R"(\\[tbnrf\\"'])";
```

Comment:
The first of those lines is just a reformatting.
In the second line their were two backslashes missing.